.github/workflows/package.yml

```
@@ -22,10 +22,10 @@ jobs:
     - name: Checkout code 
       uses: actions/checkout@v2
       
-    - name: Setup JDK 8
+    - name: Setup JDK 17
       uses: actions/setup-java@v2
```

actionlint (1.7.3)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)